Updated DwC-MIxS mapping

raissameyer opened the pull request:
This [PR](https://github.com/mapping-commons/sssom/pull/49) and [issue](https://github.com/mapping-commons/sssom/issues/46) in the SSSOM repo showed that - by following the old SSSOM guidance - we have sometimes used the SKOS predicates for narrow match and broad match incorrectly. 

In issue #77 we reviewed the mapping predicates. 

In this PR we will update our mapping files to fix this. 

## Updates for v2.0.0 of the mapping

### Mapping predicates
- value for `predicate_id` updated to `skos:narrowMatch` for the terms listed here: https://github.com/tdwg/gbwg/issues/77#issuecomment-995806725
- value for `predicate_id` updated to `skos:broadMatch` for the terms listed here: https://github.com/tdwg/gbwg/issues/77#issuecomment-995840816
- value for `syntax_predicate_id` updated to `skos:broadMatch` for the terms listed here: https://github.com/tdwg/gbwg/issues/77#issuecomment-997932073, https://github.com/tdwg/gbwg/issues/77#issuecomment-998646738

### Associated metadata
- values for `mapping_date`/`syntax_mapping_date` and `creator_id` were updated to reflect the updates in the mapping predicates

### Additional metadata
- v2.0.0 additionally includes a confidence value `confidence`/`syntax_confidence` for the mapping between 0 and 1 (0 no confidence, 1 100% confidence; see [The SSSOM 5-Star System For Mappings](https://mapping-commons.github.io/sssom/spec/#the-sssom-5-star-system-for-mappings)). 

### Removed metadata
- DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv no longer includes the additional column on `predicate_label` as there was no clear guidance on what those are for skos:narrowMatch and skos:broadMatch.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:country	Country	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:country	dwcversion:country-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	the MIxS term includes both seas and countries, while DwC only refers to the country
+dwc:waterBody	Water Body	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:waterBody	dwcversion:waterBody-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	the MIxS term includes both seas and countries, while DwC only refers to the sea.
+dwc:higherGeography	Higher Geography	https://dwc.tdwg.org/list/#dcterms_Location	skos:exactMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:higherGeography	dwcversion:higherGeography-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-02-09	0.7	DwC term encompasses any combination of geographic region names, while the MIxS term may either be the name of a country or a s ...
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	Events are semantically broader than sampling events (e.g. they could include observations too)
```
Mapping looks good following logic in https://github.com/tdwg/gbwg/issues/77 xref discussion in #17

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:broadMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	0.7	The range of the MIxS examples is [free living|parasitism|commensalism|symbiotic|m ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000002	samp_collect_device	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. Bot ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. The ob ...
+dwc:sampleSizeValue	Sample Size Value	http://rs.tdwg.org/dwc/terms/Event	skos:broadMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:sampleSizeValue	dwcversion:sampleSizeValue-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:sampleSizeValue requires the units of the measure to be given in dwc:sampleSizeUnit.
```
Looks fine - this and sampleSizeUnit are semantically (and syntactically) wrapped up in MIxS:samp_size

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. The ob ...
+dwc:sampleSizeValue	Sample Size Value	http://rs.tdwg.org/dwc/terms/Event	skos:broadMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:sampleSizeValue	dwcversion:sampleSizeValue-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:sampleSizeValue requires the units of the measure to be given in dwc:sampleSizeUnit.
+dwc:sampleSizeUnit	Sample Size Unit	http://rs.tdwg.org/dwc/terms/Event	skos:broadMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:sampleSizeUnit	dwcversion:sampleSizeUnit-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:sampleSizeUnit  must accompany the measure given in dwc:sampleSizeValue.
+dcterms:references	References	https://dwc.tdwg.org/list/#dcterms_references	skos:closeMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dcterms:references	dcversion:#references-003	MIXSversion:mixs_v5.xlsx	5	2021-02-09	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the MIxS issue tracker: https://github. ...
+dwc:associatedMedia	Associated Media	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:closeMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedMedia	dwcversion:associatedMedia-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-12	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the MIxS issue tracker: ...
+dwc:associatedOccurrences	Associated Occurrences	https://dwc.tdwg.org/list/#dwc_Organism	skos:closeMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOccurrences	dwcversion:associatedOccurrences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on t ...
+dwc:associatedOrganisms	Associated Organisms	https://dwc.tdwg.org/list/#dwc_Organism	skos:closeMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the MIxS  ...
+dwc:associatedReferences	Associated References	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:closeMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedReferences	dwcversion:associatedReferences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the ...
+dwc:associatedSequences	Associated Sequences	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:broadMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedSequences	dwcversion:associatedSequences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the MIx ...
+dwc:associatedTaxa	Associated Taxa	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:broadMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedTaxa	dwcversion:associatedTaxa-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	0.5	This mapping is a guess, given that the MIxS url term does not have a definition provided. Current conversation on the MIxS issue tracker: htt ...
```
These look fine as they all reference external identifiers. The switch to broadMatch is justified as noted in the comments.

pbuttigieg reviewed (changes requested):
Semantic mapping reviewed in consultation with @raissameyer

pbuttigieg commented:
The review of the semantic mapping (following the update of the SSSOM guidance and @raissameyer's implementation thereof) is done above. 

You'll notice that this was done in a line-by-line fashion as we think this should be the norm for a mapping file that links standards organisations: very thorough, with approvals explicit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+#  - orcid: https://orcid.org/
+#  - skos: http://www.w3.org/2004/02/skos/core
+subject_id	subject_label	subject_category	predicate_id	object_id	object_label	object_category	match_type	creator_id	subject_source	subject_source_version	object_source	object_source_version	mapping_date	confidence	comment
+dwc:verbatimCoordinates	Verbatim Coordinates	https://dwc.tdwg.org/list/#dcterms_Location	skos:narrowMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimCoordinates	dwcversion:verbatimCoordinates-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	'original spatial coordinates of the location' is semantically broader than 'latitude and longitude'. Additionally, t ...
```
Thank you, done.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:decimalLongitude	Decimal Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:decimalLongitude	dwcversion:decimalLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the longitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimLatitude	Verbatim Latitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLatitude	dwcversion:verbatimLatitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the latitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimLongitude	Verbatim Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLongitude	dwcversion:verbatimLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the longitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimDepth	Verbatim Depth	https://dwc.tdwg.org/list/#dcterms_Location	skos:narrowMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:verbatimDepth	dwcversion:verbatimDepth-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	In Darwin Core, according to best practices, depth is measured in water from an air-water surface, while in other m ...
```
Thank you, I will update the mapping predicate to skos:exactMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:verbatimLatitude	Verbatim Latitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLatitude	dwcversion:verbatimLatitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the latitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimLongitude	Verbatim Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLongitude	dwcversion:verbatimLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the longitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimDepth	Verbatim Depth	https://dwc.tdwg.org/list/#dcterms_Location	skos:narrowMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:verbatimDepth	dwcversion:verbatimDepth-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	In Darwin Core, according to best practices, depth is measured in water from an air-water surface, while in other m ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upp ...
```
Thank you, I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:verbatimLongitude	Verbatim Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLongitude	dwcversion:verbatimLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the longitude (narrower), while MIxS expects both latitude and longitude (broader)
+dwc:verbatimDepth	Verbatim Depth	https://dwc.tdwg.org/list/#dcterms_Location	skos:narrowMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:verbatimDepth	dwcversion:verbatimDepth-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	In Darwin Core, according to best practices, depth is measured in water from an air-water surface, while in other m ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upp ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the low ...
```
Thank you, as discussed I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the low ...
+dwc:minimumDepthInMeters	Minimum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumDepthInMeters	dwcversion:minimumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term specifically addresses a range of depths, for which this term gives the minimum depth, whereas the MI ...
+dwc:maximumDepthInMeters	Maximum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumDepthInMeters	dwcversion:maximumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term specifically addresses a range of depths, for which this term gives the maximum depth, whereas the MI ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower altitude li ...
```
Thank you, as discussed I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:minimumDepthInMeters	Minimum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumDepthInMeters	dwcversion:minimumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term specifically addresses a range of depths, for which this term gives the minimum depth, whereas the MI ...
+dwc:maximumDepthInMeters	Maximum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumDepthInMeters	dwcversion:maximumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term specifically addresses a range of depths, for which this term gives the maximum depth, whereas the MI ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower altitude li ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upper altitude li ...
```
Thank you, as discussed I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:maximumDepthInMeters	Maximum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumDepthInMeters	dwcversion:maximumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term specifically addresses a range of depths, for which this term gives the maximum depth, whereas the MI ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower altitude li ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upper altitude li ...
+dwc:verbatimElevation	Verbatim Elevation	https://dwc.tdwg.org/list/#dcterms_Location	skos:exactMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimElevation	dwcversion:verbatimElevation-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-02-09	1	
```
Will update the comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower altitude li ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upper altitude li ...
+dwc:verbatimElevation	Verbatim Elevation	https://dwc.tdwg.org/list/#dcterms_Location	skos:exactMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimElevation	dwcversion:verbatimElevation-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-02-09	1	
+dwc:minimumElevationInMeters	Minimum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumElevationInMeters	dwcversion:minimumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower elevation limit in meter (narrower), while MIxS expects a single elevati ...
```
Thank you, as discussed I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upper altitude li ...
+dwc:verbatimElevation	Verbatim Elevation	https://dwc.tdwg.org/list/#dcterms_Location	skos:exactMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimElevation	dwcversion:verbatimElevation-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-02-09	1	
+dwc:minimumElevationInMeters	Minimum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumElevationInMeters	dwcversion:minimumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the lower elevation limit in meter (narrower), while MIxS expects a single elevati ...
+dwc:maximumElevationInMeters	Maximum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumElevationInMeters	dwcversion:maximumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the upper elevation limit in meter (narrower), while MIxS expects a single elevati ...
```
Thank you, as discussed I will update the mapping predicate to skos:narrowMatch and update the associated comment in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	Events are semantically broader than sampling events (e.g. they could include observations too)
+dwc:materialSampleID	Material Sample ID	http://rs.tdwg.org/dwc/terms/MaterialSample	skos:exactMatch	MIXS:0000026	source_mat_id	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:materialSampleID	dwcversion:materialSampleID-2020-10-28	MIXSversion:mixs_v5.xlsx	5	2021-02-09	1	
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:broadMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	0.7	The range of the MIxS examples is [free living|parasitism|commensalism|symbiotic|m ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000002	samp_collect_device	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. Bot ...
```
Thanks, will update the mapping predicate and comment accordingly.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSemantic-2.0.0.sssom.tsv:
```diff
@@ -0,0 +1,75 @@
+dwc:materialSampleID	Material Sample ID	http://rs.tdwg.org/dwc/terms/MaterialSample	skos:exactMatch	MIXS:0000026	source_mat_id	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:materialSampleID	dwcversion:materialSampleID-2020-10-28	MIXSversion:mixs_v5.xlsx	5	2021-02-09	1	
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:broadMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	0.7	The range of the MIxS examples is [free living|parasitism|commensalism|symbiotic|m ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000002	samp_collect_device	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. Bot ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:narrowMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	dwc:samplingProtocol covers both of the concepts samp_collect_device and samp_collect_process. The ob ...
```
Thanks, will update the comment with the next commit.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:maximumElevationInMeters	Maximum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumElevationInMeters	dwcversion:maximumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:country	Country	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:country	dwcversion:country-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC example only expects one word, while MIxS expects two words (country or sea name + specific region name). Additionally MIxS expects terms from an ontolo ...
+dwc:waterBody	Water Body	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:waterBody	dwcversion:waterBody-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-05-12	1	The DwC example only expects one sea identifier, while MIxS expects an additional word (country or sea name + specific region name). Additionally, MIxS ...
+dwc:higherGeography	Higher Geography	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:higherGeography	dwcversion:higherGeography-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list (of undefined length) of values separated by a vertical bar space  '|', while MIxS expects two words (countr ...
```
Related seems to work, syntactically: both are string lists with some special characters that need not overlap. The controlled vocabs in MIxS don't really impact syntax.

DwC guidance:
>Recommended best practice is to separate the values in a list with space vertical bar space ( | ), with terms in order from least specific to most specific.

DwC examples (extract, that's the actual example of the syntax):
> South America | Argentina | Patagonia | Parque Nacional Nahuel Huapi | Neuquén | Los Lagos

MIxS guidance:
> The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)

Expected:
>country or sea name (INSDC or GAZ);region(GAZ);specific location name	

value syntax
> {term};{term};{text}

Terms coming from controlled vocabs, with text being uncontrolled

pbuttigieg replied:
Perhaps update the comment to focus exclusively on the syntax according to our rubric noted in this review.

raissameyer replied:
Thank you. Comment, with a link to the rubric set up above, updated in the next commit.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:country	Country	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:country	dwcversion:country-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC example only expects one word, while MIxS expects two words (country or sea name + specific region name). Additionally MIxS expects terms from an ontolo ...
+dwc:waterBody	Water Body	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:waterBody	dwcversion:waterBody-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-05-12	1	The DwC example only expects one sea identifier, while MIxS expects an additional word (country or sea name + specific region name). Additionally, MIxS ...
+dwc:higherGeography	Higher Geography	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:higherGeography	dwcversion:higherGeography-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list (of undefined length) of values separated by a vertical bar space  '|', while MIxS expects two words (countr ...
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:exactMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC requests compliance with ISO 8601-1:2019, so does MIxS
```
exact is fine, but keep in mind both allow non ISO entries (they just recommend using them). This should be stated in the comment and perhaps we can encourage stricter recommendations to both groups.

raissameyer replied:
Thank you for noting this. Captured the fact that both only note ISO entries as a recommendation in the comment.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:waterBody	Water Body	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:waterBody	dwcversion:waterBody-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-05-12	1	The DwC example only expects one sea identifier, while MIxS expects an additional word (country or sea name + specific region name). Additionally, MIxS ...
+dwc:higherGeography	Higher Geography	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:higherGeography	dwcversion:higherGeography-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list (of undefined length) of values separated by a vertical bar space  '|', while MIxS expects two words (countr ...
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:exactMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC requests compliance with ISO 8601-1:2019, so does MIxS
+dwc:materialSampleID	Material Sample ID	http://rs.tdwg.org/dwc/terms/MaterialSample	skos:exactMatch	MIXS:0000026	source_mat_id	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:materialSampleID	dwcversion:materialSampleID-2020-10-28	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	Both expect a mix of letters, numbers, etc.
```
Both expect PIDs of a sort (undefined / unrestricted as to what kind, that should be noted in the comment) and the universe of their syntax is overlapping to the extent of exactness. Also, the MIxS term references the DwC term, which should also be noted in the comment.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:higherGeography	Higher Geography	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:higherGeography	dwcversion:higherGeography-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list (of undefined length) of values separated by a vertical bar space  '|', while MIxS expects two words (countr ...
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:exactMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC requests compliance with ISO 8601-1:2019, so does MIxS
+dwc:materialSampleID	Material Sample ID	http://rs.tdwg.org/dwc/terms/MaterialSample	skos:exactMatch	MIXS:0000026	source_mat_id	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:materialSampleID	dwcversion:materialSampleID-2020-10-28	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	Both expect a mix of letters, numbers, etc.
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:relatedMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list of terms and corresponding identifiers, while MIxS only expects a list of terms
```
This is a close candidate for exact match, as both fields accept numbers, letters, and special characters (think candidate phyla names) and, according to the descriptions, also contain structured and paired lists. But the syntax is different enough that we can move from relatedMatch.

Note that DwC expects quoted strings - this may be enough to consider non-exact, but it's borderline as it's "just" the way it structures its lists values. 

We should have a more descriptive comment about the syntactic overlap (characters and structure)

Note the MIxS description diverges from the examples of usage, more here: https://github.com/GenomicsStandardsConsortium/mixs/issues/168

raissameyer replied:
Thanks. Comment updated in the next commit.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:relatedMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list of terms and corresponding identifiers, while MIxS only expects a list of terms
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000002	samp_collect_device	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects the name of, reference to, or description (which could also include a URI, IRI, etc.), while MIxS only expects a  ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects the name of, reference to, or description (which could also include a URI, IRI, etc.), while MIxS expects terms f:he ...
+dwc:sampleSizeValue	Sample Size Value	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeValue	dwcversion:sampleSizeValue-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {float}
+dwc:sampleSizeUnit	Sample Size Unit	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeUnit	dwcversion:sampleSizeUnit-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {unit}
```
Would these not be broadMatch cases? MIxS having more (undesirable) syntactic complexity than DwC?

xref: https://github.com/tdwg/gbwg/issues/54#issuecomment-821155504

raissameyer replied:
Thank you for catching this. Updated in next commit.

pbuttigieg commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:eventDate	Event Date	http://rs.tdwg.org/dwc/terms/Event	skos:exactMatch	MIXS:0000011	collection_date	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:eventDate	dwcversion:eventDate-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC requests compliance with ISO 8601-1:2019, so does MIxS
+dwc:materialSampleID	Material Sample ID	http://rs.tdwg.org/dwc/terms/MaterialSample	skos:exactMatch	MIXS:0000026	source_mat_id	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:materialSampleID	dwcversion:materialSampleID-2020-10-28	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	Both expect a mix of letters, numbers, etc.
+dwc:associatedOrganisms	Associated Organisms	http://rs.tdwg.org/dwc/terms/Organism	skos:relatedMatch	MIXS:0000028	biotic_relationship	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects a list of terms and corresponding identifiers, while MIxS only expects a list of terms
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000002	samp_collect_device	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects the name of, reference to, or description (which could also include a URI, IRI, etc.), while MIxS only expects a  ...
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects the name of, reference to, or description (which could also include a URI, IRI, etc.), while MIxS expects terms f:he ...
```
These may be cases of DwC narrowMatch to MIxS as the DwC description calls for a higher diversity of strings. However, the MIxS term "label" and its description can also be read as quite unbounded. Either the related or narrowMatch would be fine in IMO, but we should add some more qualification to the comment.

raissameyer replied:
Thank you. 

I would argue to stick with relatedMatch, as the MIxS term and its expected syntax is too loosely defined to narrow down the expected diversity of strings. Indeed, the MIxS field is on a similar level as the verbatim fields above, which we decided to related to other terms with `relatedMatch` as they can have free text (or any other string-based value) as syntax.

pbuttigieg reviewed (changes requested):
Syntax review in for your consideration @raissameyer - thanks for the thorough work, almost there!

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:verbatimElevation	Verbatim Elevation	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimElevation	dwcversion:verbatimElevation-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC term expects verbatim input (so anything really), whereas MIxS expects {float} {unit} in meter
+dwc:minimumElevationInMeters	Minimum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumElevationInMeters	dwcversion:minimumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:maximumElevationInMeters	Maximum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumElevationInMeters	dwcversion:maximumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:country	Country	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:country	dwcversion:country-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC example only expects one word, while MIxS expects two words (country or sea name + specific region name). Additionally MIxS expects terms from an ontolo ...
```
Thank you, comment and mapping will be updated in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:minimumElevationInMeters	Minimum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumElevationInMeters	dwcversion:minimumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:maximumElevationInMeters	Maximum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumElevationInMeters	dwcversion:maximumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:country	Country	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:country	dwcversion:country-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC example only expects one word, while MIxS expects two words (country or sea name + specific region name). Additionally MIxS expects terms from an ontolo ...
+dwc:waterBody	Water Body	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000010	geo_loc_name	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:waterBody	dwcversion:waterBody-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-05-12	1	The DwC example only expects one sea identifier, while MIxS expects an additional word (country or sea name + specific region name). Additionally, MIxS ...
```
Thank you, comment and mapping will be updated in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:decimalLongitude	Decimal Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:decimalLongitude	dwcversion:decimalLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	The DwC term expects decimal degrees, and so does MIxS - however, the DwC term expects one value, while MIxS expects two
+dwc:verbatimLatitude	Verbatim Latitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLatitude	dwcversion:verbatimLatitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-03-30	1	MIxS expects two input values in decimal degrees, while DwC only expects one as verbatim (so anything really)
+dwc:verbatimLongitude	Verbatim Longitude	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000009	lat_lon	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:verbatimLongitude	dwcversion:verbatimLongitude-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-03-30	1	MIxS expects two input values in decimal degrees, while DwC only expects one as verbatim (so anything really)
+dwc:verbatimDepth	Verbatim Depth	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimDepth	dwcversion:verbatimDepth-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC term expects a verbatim input (so anything really), while MIxS expects a {float} {unit} entry in meters
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The example offered by DwC does not include the unit ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The example offered by DwC does not include the unit ...
+dwc:minimumDepthInMeters	Minimum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumDepthInMeters	dwcversion:minimumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be added {float} {un ...
+dwc:maximumDepthInMeters	Maximum Depth In Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000018	depth	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumDepthInMeters	dwcversion:maximumDepthInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be added {float} {un ...
+dwc:minimumDistanceAboveSurfaceInMeters	Minimum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:minimumDistanceAboveSurfaceInMeters	dwcversion:minimumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, wh ...
+dwc:maximumDistanceAboveSurfaceInMeters	Maximum Distance Above Surface in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch		alt	MIxS core	HumanCurated	orcid:0000-0003-1144-0290 | orcid:0000-0002-2996-719X	dwc:maximumDistanceAboveSurfaceInMeters	dwcversion:maximumDistanceAboveSurfaceInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, wh ...
+dwc:verbatimElevation	Verbatim Elevation	https://dwc.tdwg.org/list/#dcterms_Location	skos:relatedMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:verbatimElevation	dwcversion:verbatimElevation-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	The DwC term expects verbatim input (so anything really), whereas MIxS expects {float} {unit} in meter
+dwc:minimumElevationInMeters	Minimum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:minimumElevationInMeters	dwcversion:minimumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
+dwc:maximumElevationInMeters	Maximum Elevation in Meters	https://dwc.tdwg.org/list/#dcterms_Location	skos:broadMatch	MIXS:0000093	elev	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:maximumElevationInMeters	dwcversion:maximumElevationInMeters-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-12-20	1	DwC expects only the value {float}, whereas the MIxS syntax expects the value and the unit to be ad ...
```
Comments will be updated in the next commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dcterms:references	References	https://dwc.tdwg.org/list/#dcterms_references	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dcterms:references	dcversion:#references-003	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	
+dwc:associatedMedia	Associated Media	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedMedia	dwcversion:associatedMedia-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
+dwc:associatedOccurrences	Associated Occurrences	https://dwc.tdwg.org/list/#dwc_Organism	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOccurrences	dwcversion:associatedOccurrences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
+dwc:associatedOrganisms	Associated Organisms	https://dwc.tdwg.org/list/#dwc_Organism	skos:relatedMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	The DwC term expects a link as well as text about the type of association, whereas MIxS only expects a url
```
Thank you. Updated in the coming commit.

raissameyer replied:
Thank you. Updated in the coming commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:associatedOccurrences	Associated Occurrences	https://dwc.tdwg.org/list/#dwc_Organism	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOccurrences	dwcversion:associatedOccurrences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
+dwc:associatedOrganisms	Associated Organisms	https://dwc.tdwg.org/list/#dwc_Organism	skos:relatedMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOrganisms	dwcversion:associatedOrganisms-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	The DwC term expects a link as well as text about the type of association, whereas MIxS only expects a url
+dwc:associatedReferences	Associated References	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:relatedMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedReferences	dwcversion:associatedReferences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	The DwC would expect a comma separated list of citations, URIs, etc.
+dwc:associatedSequences	Associated Sequences	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedSequences	dwcversion:associatedSequences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
```
Thank you. Updated in the coming commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:sampleSizeUnit	Sample Size Unit	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeUnit	dwcversion:sampleSizeUnit-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {unit}
+dcterms:references	References	https://dwc.tdwg.org/list/#dcterms_references	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dcterms:references	dcversion:#references-003	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	
+dwc:associatedMedia	Associated Media	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedMedia	dwcversion:associatedMedia-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
+dwc:associatedOccurrences	Associated Occurrences	https://dwc.tdwg.org/list/#dwc_Organism	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedOccurrences	dwcversion:associatedOccurrences-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
```
Thank you. Updated in the coming commit.

raissameyer replied:
Thank you. Updated in the coming commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:sampleSizeValue	Sample Size Value	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeValue	dwcversion:sampleSizeValue-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {float}
+dwc:sampleSizeUnit	Sample Size Unit	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeUnit	dwcversion:sampleSizeUnit-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {unit}
+dcterms:references	References	https://dwc.tdwg.org/list/#dcterms_references	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dcterms:references	dcversion:#references-003	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	
+dwc:associatedMedia	Associated Media	https://dwc.tdwg.org/list/#dwc_Occurrence	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0002-2996-719X	dwc:associatedMedia	dwcversion:associatedMedia-2020-08-12	MIXSversion:mixs_v5.xlsx	5	2021-04-12	1	
```
Thank you. Updated in the coming commit.

raissameyer replied:
Thank you. Updated in the coming commit.

raissameyer replied:
Thank you. Updated in the coming commit.

raissameyer commented on dwc-mixs/mapping/DwC-MIxS_mappingSyntactic-2.0.0.sssom.tsv:
```diff
@@ -40,36 +40,36 @@
+dwc:samplingProtocol	Sampling Protocol	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000016	samp_mat_process	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:samplingProtocol	dwcversion:samplingProtocol-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	DwC expects the name of, reference to, or description (which could also include a URI, IRI, etc.), while MIxS expects terms f:he ...
+dwc:sampleSizeValue	Sample Size Value	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeValue	dwcversion:sampleSizeValue-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {float}
+dwc:sampleSizeUnit	Sample Size Unit	http://rs.tdwg.org/dwc/terms/Event	skos:relatedMatch	MIXS:0000001	samp_size	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dwc:sampleSizeUnit	dwcversion:sampleSizeUnit-2017-10-06	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	MIxS expects {float} {unit}, while DwC only expects {unit}
+dcterms:references	References	https://dwc.tdwg.org/list/#dcterms_references	skos:exactMatch	MIXS:0000091	url	MIxS core	HumanCurated	orcid:0000-0003-1144-0290	dcterms:references	dcversion:#references-003	MIXSversion:mixs_v5.xlsx	5	2021-04-16	1	
```
Thank you. Updated in the coming commit.

raissameyer replied:
Thank you. Updated in the coming commit.

raissameyer commented:
Thank you for the in depth review @pbuttigieg. I have addressed the comments as discussed, and updated the semantic and syntactic mapping file accordingly. 

Please let me know if there are any further comments.

Once we are good to go, I'll go ahead and also update the Support mapping file to reflect the changes in the semantic and syntactic mapping.
